src/lightning-address.ts

```diff
@@ -75,31 +75,45 @@ export default class LightningAddress {
     );
     const json = await result.json();
 
-    await this.parseResponse(json.lnurlp, json.keysend, json.nostr);
+    await this.parseLnUrlPayResponse(json.lnurlp);
```

Comment:
why is this one async and the others aren't? did you miss something in your refactoring?

Reply:
No I haven't. Good eye. First one uses utils `await parseLnUrlPayResponse()` because `sha256` is async. Others doesn't have async code.